content/en/docs/usage/kraft_advanced.md

```diff
@@ -0,0 +1,609 @@
+  => build/hello_kvm-x86_64.dbg (with symbols)
+
+To instantiate, use: kraft run
+```
+
+### Overview of `kraft build` command
+
+```text
+Usage: kraft build [OPTIONS] [TARGET]
+
+  Builds the Unikraft application for the target architecture and platform.
+
+Options:
+  -v, --verbose               Verbose build
+  --fetch / --no-fetch        Run fetch step before build.
+  --prepare / --no-prepare    Run prepare step before build.
+  --progress / --no-progress  Show progress of build.
+  -j, --fast                  Use all CPU cores to build the application.
+  -F, --force                 Force the build of the unikernel.
+  -h, --help                  Show this message and exit.
+
+```
+
+## `kraft run`
+
+You can run your application using `kraft run`.
+You can specify the target architecture and platform using `-m, --arch` and `-p, --plat`.
+
+```
+$ kraft run -m x86_64 -p kvm
+```
+
+### Attach a file system to the application
+
+#### Use an InitRD filesystem
+
+You can add an InitRD filesystem to the application, by using the `-i, --initrd` flags.
+
+```
+$ kraft run -m x86_64 -p kvm -i archive.cpio
+```
+
+#### Use a `9pfs` filesystem
+
+You can also attach a `9pfs` filesystem to your application.
+In order to do that, you need to make sure you have set the following options in the application configuration:
+
+```text
+- CONFIG_LIBVFSCORE_ROOTFS_9PFS=y
+- CONFIG_LIBVFSCORE_ROOTDEV="fs0"
+```
+
+You can check the `kraft.yaml` file from the [Python 3 app](https://github.com/unikraft/app-python3/blob/staging/kraft.yaml#L5) to get a better idea of the required configuration.
+By default, the `kraft` application uses the name `fs0` for the device to mount the filesystem from.
+To use your application with `kraft`, you must name the shared filesystem "fs0".
+
+```
+$ mkdir fs0
+$ echo 'Hello world!' > fs0/file.txt
+$ kraft run
+[...]
+
+SeaBIOS (version 1.13.0-1ubuntu1.1)
+Booting from ROM...
+Powered by
+o.   .o       _ _               __ _
+Oo   Oo  ___ (_) | __ __  __ _ ' _) :_
+oO   oO ' _ `| | |/ /  _)' _` | |_|  _)
+oOo oOO| | | | |   (| | | (_) |  _) :_
+ OoOoO ._, ._:_:_,\_._,  .__,_:_, \___)
+                     Mimas 0.7.0~1939bce
+Hello world!
+```
+
+### Use `kraft run` to debug you application
+
+If you want to debug the application, you can launch it in a paused state by using the `-P, --paused` flags.
+You can run a GDB server for the guest on a specific port using the `-g, --gdb PORT` flags.
+
+```
+$ kraft run -P -g 1234
+```
+
+To connect the GDB to the created server, you can run
+
+```
+$ gdb --eval-command="target remote :1234" build/application-image.dbg
+[...]
+Reading symbols from build/application-image.dbg
+Remote debugging using :1234
+```
+
+You can set the debug information level from the `Build options --> Debug information level` screen in `kraft menuconfig`.
+For more informations about debugging with Unikraft, please check the [debugging section](docs/develop/debugging).
+
+### Overview of `kraft run` command
+
```

Comment:
Add some text here, saying something like "You can get a full list of features by running `kraft run` with no arguments:"